arctic/serialization/numpy_records.py

```diff
@@ -11,24 +11,6 @@
 DTN64_DTYPE = 'datetime64[ns]'
```

Comment:
Perhaps docstring that this module is used by the existing version store's _pandas_ndarray_store.